### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by ark2038**
minor updates

### CODE REVIEW

**Comment by albertog78** [comment]
Add fixes related to the RPC tests https://github.com/zcash/zcash/compare/v2.1.1...v2.1.1-1?diff=split#diff-35a5a0a43a0f4c6dd4bf20eb27065187R131

File: src/init.cpp [hunk 1]
```diff
@@ -355,6 +356,8 @@ std::string HelpMessage(HelpMessageMode mode)
...
     strUsage += HelpMessageOpt("-loadblock=<file>", _("Imports blocks from external blk000??.dat file") + " " + _("on startup"));
     strUsage += HelpMessageOpt("-maxorphantx=<n>", strprintf(_("Keep at most <n> unconnectable transactions in memory (default: %u)"), DEFAULT_MAX_ORPHAN_TRANSACTIONS));
+    strUsage += HelpMessageOpt("-maxtimeadjustment=<n>", strprintf(_("Maximum allowed median peer time offset adjustment, in seconds. Local perspective of time may be influenced by peers forward or backward by this amount. (default: %u seconds, maximum: %u seconds)"),
+    		DEFAULT_MAX_TIME_ADJUSTMENT, LIMIT_MAX_TIME_ADJUSTMENT));
     strUsage += HelpMessageOpt("-mempooltxinputlimit=<n>", _("Set the maximum number of transparent inputs in a transaction that the mempool will accept (default: 0 = no limit applied)"));
     strUsage += HelpMessageOpt("-par=<n>", strprintf(_("Set the number of script verification threads (%u to %d, 0 = auto, <0 = leave that many cores free, default: %d)"),
         -GetNumCores(), MAX_SCRIPTCHECK_THREADS, DEFAULT_SCRIPTCHECK_THREADS));
```
Comment (albertog78) [inline src/init.cpp:363]: add the following check on maxTimeAdjustment:
int64_t nMaxTimeAdjustment = GetArg("-maxtimeadjustment", DEFAULT_MAX_TIME_ADJUSTMENT);
    if (nMaxTimeAdjustment < 0 || nMaxTimeAdjustment > LIMIT_MAX_TIME_ADJUSTMENT) {
        return InitError(strprintf(_("-maxtimeadjustment must be in the range 0 to %u seconds"), LIMIT_MAX_TIME_ADJUSTMENT));
    }

**Review by albertog78 — CHANGES_REQUESTED** [review]

File: src/init.cpp (hunk 1 above)
Comment (ark2038) [reply src/init.cpp:363]: done

Comment (ark2038) [reply src/init.cpp:363]: done

**Comment by ark2038** [comment]
> Add fixes related to the RPC tests https://github.com/zcash/zcash/compare/v2.1.1...v2.1.1-1?diff=split#diff-35a5a0a43a0f4c6dd4bf20eb27065187R131

**Review by albertog78 — APPROVED** [review]

**Comment by pierstab** [comment]
added to gtest/forkmanager the futuretimestamo activation height test

**Comment by ark2038** [comment]
added to gtest/forkmanager the futuretimestamp activation height test

**Review by cronicc — APPROVED** [review]

**Comment by cronicc** [comment]
Pulled into https://github.com/ZencashOfficial/zen/tree/timeupdate

**Pull request closed without merging**